java-frontend/src/main/java/org/sonar/java/model/VisitorsBridge.java

```diff
@@ -85,11 +86,13 @@ public VisitorsBridge(JavaFileScanner visitor) {
   }
 
   @VisibleForTesting
-  public VisitorsBridge(Iterable visitors, List<File> projectClasspath, @Nullable SonarComponents sonarComponents) {
+  public VisitorsBridge(Iterable<? extends JavaCheck> visitors, List<File> projectClasspath,
```

Comment:
I don't see why this type needs to be specified. it's seems unrelated to your change,

Reply:
it's not strictly related to this PR, but I wanted to remove usage of raw type to benefit from compiler checking the correctness of types.